rllib/contrib/bandits/agents/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ray.rllib.contrib.bandits.agents.linear import LinTSTrainer, LinUCBTrainer
+
+__all__ = ["LinTSTrainer", "LinUCBTrainer"]
```

Comment:
Can you add these to contrib/registry.py so that it can be run from the command line?